examples/dswx_ni_sample_runconfig-v4.0.0-er.2.0.yaml

```diff
@@ -182,6 +182,8 @@ RunConfig:
             # Separate runconfig file for algorithm parameters
             algorithm_parameters: /home/dswx_user/input_dir/algorithm_parameter_ni.yaml
 
+            glad_classification_file: /home/dswx_user/input_dir/glad_classification.tif
```

Comment:
Similar to the changes you made to the RunConfig used with the integration test, this field needs to appear in the PGE section of the config as well, within `AncillaryFileMap`